js/libs/form.js

```diff
@@ -364,7 +364,9 @@ Fliplet.Widget.instance('form-builder', function(data) {
             return operation.then(function () {
               Fliplet.Navigate.to(data.linkAction);
             }).catch(function (err) {
-              alert(err.description || err.message || err.description || err.reason || err);
+              Fliplet.Modal.alert({
+                message: err.description || err.message || err.description || err.reason || err
```

Comment:
@tonytlwu I think we can replace this with `message: Fliplet.parseError(err)`, am I right?

Reply:
Correct

Reply:
fixed